spec/puppet-server-lib/puppet/jvm/puppet_config_spec.rb

```diff
@@ -21,13 +21,6 @@
       end
     end
 
-    describe '(SERVER-410) Puppet[:always_cache_features]' do
-      subject { Puppet[:always_cache_features] }
-      it 'is true for increased performance in puppet-server' do
-        expect(subject).to eq(true)
-      end
-    end
-
```

Comment:
Oh, I see, and we are setting `always_retry_features` to be `false`.  Does that mean we're losing a performance improvement from caching?  Or am I starting to ask questions with answers that I don't really want to know?

... or ...

Is it the case that `always_retry_plugins = false` implies `always_cache_features = true`?  😕